app/models/planting.rb

```diff
@@ -113,6 +113,20 @@ def percentage_grown
     100
   end
 
+  def update_harvest_days
+    harvests_with_dates = harvests.where.not(harvested_at: nil)
+    first_harvest = nil
+    last_harvest = nil
+    if planted_at.present? && harvests_with_dates.size.positive?
+      # how long ago was this planted
+      first_harvest = (harvests_with_dates.order(harvested_at: :asc).limit(1).first.harvested_at - planted_at).to_i
```

Comment:
AFAICT [first implies limit(1)](http://api.rubyonrails.org/classes/ActiveRecord/FinderMethods.html#method-i-first), so we don't need both. Alternatively, this could be `harvests_with_dates.minimum(:harvested_at)` (and `maximum` below).

Reply:
i think when i did .last() it pulled all the records. So i added .limit(1). 

I can do some more checking, but i did find (maybe in older rails) that ActiveRecord needed more clues to work out to only pull one record.

Reply:
Weird. I think `minimum` is clearer, anyway.

Reply:
I've changed to minimum over in the other PR